hips/tiles/tile.py

```diff
@@ -0,0 +1,131 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.utils.data import get_pkg_data_filename
```

Comment:
No, it's not used in the constructor (only in the docstring example, which is independent).
Try removing this import line and you'll see that everything still works (I'm pretty sure, but not 100%, I didn't try myself).